doc/api/debugger.md

```diff
@@ -187,12 +187,15 @@ flag instead of `--inspect`.
 $ node --inspect index.js
 Debugger listening on 127.0.0.1:9229.
 To start debugging, open the following URL in Chrome:
-    chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=127.0.0.1:9229/dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
+    chrome-devtools://devtools/bundled/js_app.html?experiments=true&v8only=true&ws=127.0.0.1:9229/dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
 ```
 
 (In the example above, the UUID dc9010dd-f8b8-4ac5-a510-c1a114ec7d29
 at the end of the URL is generated on the fly, it varies in different
 debugging sessions.)
 
+> If the Chrome browser is older than <strong>66.0.3345.0</strong>,
```

Comment:
nit: There's no need for the leading `> ` here.

Reply:
Also no need for `<strong>`. Would prefer markdown for emphasis, but honestly, no emphasis is needed here.

Reply:
Done.